rotary/rules/rotary.smk

```diff
@@ -5,14 +5,32 @@ import os
 import sys
 import pandas as pd
 import itertools
+import csv
 from snakemake.utils import min_version
 
+
 VERSION_POLYPOLISH="0.5.0"
 VERSION_DFAST="1.2.18"
 VERSION_EGGNOG="5.0.0" # See http://eggnog5.embl.de/#/app/downloads
 START_HMM_NAME = os.path.splitext(os.path.basename(config.get("hmm_url")))[0]
 VERSION_GTDB_COMPLETE= "214.1" # See https://data.gtdb.ecogenomic.org/releases/
 VERSION_GTDB_MAIN=VERSION_GTDB_COMPLETE.split('.')[0] # Remove subversion
+DB_DIR_PATH = config.get('db_dir')
+
+samples = []
```

Comment:
I am still not that familiar with best practices for complex snakefiles. I wonder, though, it it might improve readability to put this sample parser into a function and then call the function? This isn't necessary with a single code block at this point, but it is something we should consider down the road.

Reply:
@jmtsuji Done.